examples/hosts/app-integration/schema-upgrade/package.json

```diff
@@ -3,7 +3,11 @@
   "version": "0.58.2000",
```

Comment:
Version here (and also dependencies) will need to update to match main (currently 0.58.3000)